vcstool/commands/help.py

```diff
@@ -30,15 +30,16 @@ def main(args=None, stdout=None, stderr=None):
         print('The available VCS clients are:')
         for client in vcstool_clients:
             print('  ' + client.type)
-        return 0
-
-    if ns.commands:
+    elif ns.commands:
         print(' '.join([cmd.command for cmd in vcstool_commands]))
-        return 0
+    elif ns.commands_descriptions:
+        print('\n'.join(['{}\t{}'.format(cmd.command, cmd.help)
+                         for cmd in vcstool_commands]))
+    else:
+        # output detailed command list
+        parser = get_parser_with_command_only()
+        parser.print_help()
 
-    # output detailed command list
-    parser = get_parser_with_command_only()
-    parser.print_help()
```

Comment:
Please keep the current structure of the code (`if ...: ...; return 0`) which should minimize the changes.

Reply:
I thought of refactoring this since I believe the `if .. elif .. else` conditional is easier to read; I'll revert it

Reply:
done

Reply:
> I thought of refactoring this since I believe the if .. elif .. else conditional is easier to read;

Certainly subjective :wink: Imo it is easier to see that when an `if args.xxx:` block contains a return I don't have to look further down if it does anything else.

Reply:
It looks like that the updated patch doesn't handle `args.commands_descriptions` anymore?

Reply:
Ouch, rushed it..

Reply:
No problem. Thanks for the quick updates!